features/dd-sdk-android-session-replay-compose/src/main/kotlin/com/datadog/android/sessionreplay/compose/internal/mappers/semantics/ImageSemanticsNodeMapper.kt

```diff
@@ -82,7 +86,7 @@ internal class ImageSemanticsNodeMapper(
 
         val newBitmap = bitmap?.let {
             @Suppress("UnsafeThirdPartyFunctionCall") // isMutable is always false
-            it.copy(it.config, false)
+            it.copy(Bitmap.Config.ARGB_8888, false)
```

Comment:
what is the reason for this change?

Reply:
When testing with some other applications, I found that due to the original config of bitmap, some bitmaps are empty after copying it, for example `ALPHA_8`, I need to tweak the config in order to make it work.

Here is a related question on SO:
https://stackoverflow.com/questions/9247369/alpha-8-bitmaps-and-getpixel

Reply:
From SO:

> Simplest workaround is to waste lots of memory

Not sure though if it is useful to try to record `ALPHA_8` bitmaps, because documentation says:

```
 * Each pixel is stored as a single translucency (alpha) channel.
 * This is very useful to efficiently store masks for instance.
 * No color information is stored.
 * With this configuration, each pixel requires 1 byte of memory.
```

So probably it is a bitmask layer, rather than something that is rendered to the user. Also memory usage will be much higher - 4x more (pixel is 1 byte vs pixel is 4 bytes in cases of `ARGB_8888`).

Maybe we should keep here a TODO item for the further investigation? Maybe it makes sense to skip recording `ALPHA_8` bitmaps (if it won't affect anything).

Reply:
theoretically yes it should be a translucency channel, but when I debug I see a bitmap like this:
![image](https://github.com/user-attachments/assets/efa0782c-1e57-4463-8560-6699d83576ce)

and its original config is ALPHA_8. With the previous code it will copy nothing, but when changing to ARGB_888, it has real data.

Reply:
okay, makes sense then if we assume that with such config pixel is always just black and some alpha is applied. Maybe then we should use `ARGB_8888` only when original config is `ALPHA_8`? And use the original config otherwise.

Reply:
I haven't test exhaustively all the config, but I will create a ticket to investigate all of them